Core/Core/View/Base/FlexibleKeyboardInputView.swift

```diff
@@ -71,19 +75,22 @@ public struct FlexibleKeyboardInputView: View {
                                         )
                                 ).padding(8)
                             Button(action: {
-                                if commentText.trimmingCharacters(in: .whitespacesAndNewlines).count > 0 {
+                                if canSend {
                                     sendText(commentText)
                                     self.commentText = ""
                                 }
                             }, label: {
                                 VStack {
-                                    commentText.trimmingCharacters(in: .whitespacesAndNewlines).count > 0
-                                    ? CoreAssets.send.swiftUIImage
-                                    : CoreAssets.sendDisabled.swiftUIImage
+                                    CoreAssets.send.swiftUIImage
+                                        .renderingMode(.template)
+                                        .foregroundStyle(Theme.Colors.accentColor)
```

Comment:
It's an action button so it would be nice if you can use `accentXColor` here.

Reply:
Done